.golangci.yml

```diff
@@ -11,6 +11,8 @@ linters:
     - errorlint
     - forbidigo
     - loggercheck
+    - staticcheck
```

Comment:
[Documentation](https://golangci-lint.run/usage/linters/#enabled-by-default) says `staticcheck` is enabled by default which makes me think this isn't necessary.

Reply:
@andyasp yes, it's not necessary per se. I just figured it's good to be explicit about enabling it.

Reply:
Either way works for me. It only jumped out at me because I didn't know why it was included and went looking.

Reply:
Removed it.